extensions-core/s3-extensions/src/test/java/io/druid/storage/s3/S3DataSegmentMoverTest.java

```diff
@@ -110,6 +110,52 @@ public void testMoveException() throws Exception
         ImmutableMap.<String, Object>of("baseKey", "targetBaseKey", "bucket", "archive")
     );
   }
+  
+  @Test
+  public void testIgnoresMissing() throws Exception
```

Comment:
Maybe a more explicit test name would be useful to avoid confusion with the following test